src/datagen/java/blusunrize/immersiveengineering/data/IEBlockTags.java

```diff
@@ -95,9 +95,17 @@ protected void addTags()
 				{
 					assert tags.ore!=null;
 					tag(tags.ore).add(IEBlocks.Metals.ORES.get(metal).get());
+					tag(tags.ore).add(IEBlocks.Metals.DEEPSLATE_ORES.get(metal).get());
```

Comment:
You should be able to just chain these two calls, so
```java
tag(tags.ore)
    .add(IEBlocks.Metals.ORES.get(metal).get())
    .add(IEBlocks.Metals.DEEPSLATE_ORES.get(metal).get());
```
That would be more readable IMO since it's obvious that both are added to the same tag.